Update enterprise SSO docs (#18836)
netdata/netdata: docs/netdata-cloud/authentication-and-authorization/enterprise-sso-authentication.md

### From Netdata Sign-up page

#### Requirements

You have to update your DNS settings by adding a TXT record with the Netdata verification code as its **Value**.

> The **Value** can be found by clicking on **DNS txt record** button, at your space settings on the **User Management** section, under **Authentication & Authorization** tab.

Reviewer: ```suggestion
The **Value** can be found by clicking the **DNS TXT record** button in your space settings under **User Management**, in the** Authentication & Authorization** tab.
```

Log into your domain provider’s website, and navigate to the DNS records section.

> Create a new TXT-record with the following specifications:
> - Value/Answer/Description: `"netdata-verification=[VERIFICATION CODE]"`
> - Name/Host/Alias: Leave this blank or type @ to include a subdomain.
> - Time to live (TTL): "86400" (this can also be inherited from the default configuration).

Reviewer: ```suggestion
Create a new TXT record with the following specifications:
- Value/Answer/Description: `"netdata-verification=[VERIFICATION CODE]"`
- Name/Host/Alias: Leave this blank or type @ to include a subdomain.
- Time to live (TTL): "86400" (this can also be inherited from the default configuration).
```
